src/Nri/Ui/Block/V6.elm

```diff
@@ -16,6 +16,8 @@ module Nri.Ui.Block.V6 exposing
 
 {-| Changes from V5:
 
+    - Remove `wordWithId` and `blankWithId` as we no longer are trying to point to words or blanks with the question box.
```

Comment:
OHHH 💡 nice, thanks

Reply:
Heads up, though, there are uses still:
<img width="710" alt="image" src="https://github.com/NoRedInk/noredink-ui/assets/8811312/230097c4-9c5a-42bf-a834-d2a4568f9677">

https://github.com/NoRedInk/NoRedInk/blob/master/lib/scaffolding/elm/src/Nri/Scaffolding/Sentence/Render.elm#L665

Reply:
Yup, but we don't use the IDs for anything (those usages should have been pulled when we switched to the `CharacterStage`).